grails-bom/build.gradle

```diff
@@ -1,108 +1,203 @@
-apply plugin: 'maven-publish'
+// Copyright 2024 the original author or authors.
+
+plugins {
+    id 'java-platform'
+    id 'maven-publish'
+}
+
+javaPlatform {
+    allowDependencies()
+}
+
+def plugins = new Properties()
+plugins.load(new StringReader(new File("$projectDir/plugins.properties").text))
+
+def profiles = new Properties()
+profiles.load(new StringReader(new File("$projectDir/profiles.properties").text))
+
+def versions = new Properties()
+versions.load(new StringReader(new File("$projectDir.parentFile/gradle.properties").text))
+
+ext {
+    // When making changes in the dependencyVersions, remember to also update the Grails BOM Documentation:
+    // https://docs.grails.org/snapshot/ref/Dependency%20Versions/Grails%20BOM.html
+    // Grails dependencies are listed in SCOTTL format:
+    // groupId:artifactId-prefixes:artifactId-suffixes:version:versionNameOverride
+    // version name = versionNameOverride ?: artifactId-prefixes[0]
+    dependenciesOverview = """\
+        com.fasterxml.jackson.core:jackson-databind::
+        com.github.ben-manes.caffeine:caffeine::
+        com.github.javaparser:javaparser-core::
+        com.h2database:h2::
+        commons-codec:commons-codec::
+        io.methvin:directory-watcher::
+        jakarta.annotation:jakarta.annotation-api::
+        jakarta.inject:jakarta.inject-api::
+        jakarta.persistence:jakarta.persistence-api::
+        jakarta.servlet:jakarta.servlet-api::
+        jakarta.xml.bind:jakarta.xml.bind-api::
+        jline:jline::
+        net.bytebuddy:byte-buddy::
+        net.java.dev.jna:jna::
+        org.apache.ant:ant:,junit:
+        org.apache.commons:commons-text::
+        org.apache.groovy:groovy:ant,cli-picocli,console,dateutil,json,macro,nio,sql,swing,templates,test,test-junit5,xml:$groovyVersion
+        org.apache.tomcat:tomcat-jdbc:::tomcat
+        org.apache.tomcat.embed:tomcat-embed-core:::tomcat
```

Comment:
`tomcat-jdbc` and `tomcat-embed-core` is already imported at version `10.1.30`. We are at `10.1.31`

Reply:
@matrei I removed `testRuntimeOnly "org.apache.tomcat.embed:tomcat-embed-logging-juli" from `grails-web-boot/build.gradle`. I am not sure what it was there for? Tests pass without it and it was the only module referring that dependency. 

I also removed tomcat-jdbc and tomcat-embed-core from bom. spring's version will be used instead.